tests/@angular_devkit/build_ng_packagr/ng-packaged/.angular-cli.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
I know these files are needed for now but we need to review the final file set for adding a library to an existing project before wrapping up.

Reply:
Should I strip the test to be a minimal sample? WOuld only be a `package.json`, a few `*.ts`/`*.html`/`*.css` files and then I'll verify the `dist` folder is created.

Reply:
Yes that would be preferable, that way the tests only include what they need and it is easier to keep them updated.